### PR TITLE
Feature jpg version

### PR DESCRIPTION
**Pull request by sethwoodworth**

### CODE REVIEW

**Comment by rdhyee** [comment]
want a review or should we just merge and see what happens?

**Comment by sethwoodworth** [comment]
Go ahead and merge it, we know it is broken.

File: .travis.yml [hunk 1]
```diff
@@ -21,11 +22,7 @@ deploy:
...
   api_key:
-    secure: YLf4rbb/f1k7+tRn2Pr8+qs/pjR9TpNF2PFKl2soe/oGSI9J9QHajzjExmKTn+tfWE17Tc4+0Hzf1/eTMvCR3IMml+gwXABLlxBMbg1hXQbnnplYQD9idVXADrJ14fbXOOzit1JsZhS9OH7pn5ccqMeCgtLkwnirT90yQlxoSgvJuUt/szE4ELnLnvNgQ5YNFJaTDPoW6OkcSyFvUrCxXklFj1XCZatYtLVQhmZqncYJcKT3b97hlmTzkeLg9by7k8D+QH96wraYKD2Kr9av61rRCUHDLAfWM3a6aV2L2vDLnly/0bd+45+e5k/K1ZEGZWXvJiUS5wy2fK+OxIoEaVghXkQa1OrqlnAKL9YmEFqxotx9CxrTURM6+7rujORpC26rwuIyX8gKKHYmBLRZZXsOmAKxGSzVaH4n1s/qVfrkYOo3pGNzkAhcoJM6OFis8vHWKr9KU2gqSPwLWKgMwKb7RktEpf/DuViYFrNB/L7VIhsVOnvjvJjiixF70JoR89k8rZz8ZDaQmt17T+zf9Q2plHsyL7mFFous13LN1kJhnVPXwjdIoivLC6T7YcGgJ3fv+aF36Ols6u6oYkgAx+aLaTxKP0D96ctdLHVc9t/YGd7E47Mye9cOZQmJc5KbCVz+dip+HrZFXNfHSXnCYQIZCLfQ4KKn0ES41ye3ND0=
+    secure: hlvS8pru/BXucO3yYc7zjUEQ1SEH+nSaQk7X8PCWYDgecrY9wE39qh6oK2dT7jH+aRhD4CmFKfKe9R1T88oPbI3UMR9NXFtJG1pO4wTTvdCGYST6Pl/KdTYkeubHSJiaJ0FbONQCtuEnpJYI474Ai6GU0aZBwI1H8W990oD0FaM=
   file: Adventures-of-Huckleberry-Finn.epub
   on:
-    repo: GITenberg/Adventures-of-Huckleberry-Finn_76
-addons:
-  apt:
-    packages:
-    - xsltproc
```
Comment (rdhyee) [inline .travis.yml:31]: @sethwoodworth why did you get rid of xsltproc?

**Comment by rdhyee** [comment]
https://github.com/GITenberg/Adventures-of-Huckleberry-Finn_76/compare/81f48b6d77e1bf7e11cd6f2006845bbd46659efa...master#diff-354f30a63fb0907d4ad57269548329e3R5 shows what was required to make travis happy and release work.   Now the question is whether the versioning actually worked. (Asking @sethwoodworth to confirm)

**Pull request merged**